operations/template/main.tf

```
@@ -1,7 +1,7 @@
 locals {
   environment_to_rs_environment_prefix_mapping = {
     dev     = "staging"
-    staging = "staging"
+    staging = "stg"
```

This should be flipped.  It's our environment on the left with RS's on the right.  So our `stg` environment should interact with RS's `staging` environment.

```suggestion
    stg = "staging"
```